### PR TITLE
Delete characters

### PR DESCRIPTION
**Pull request by kohei-s**

### CODE REVIEW

File: backend/src/test/java/de/neuefische/koheis/backend/charactercards/CharacterCardIntegrationTest.java [hunk 1]
```diff
@@ -82,31 +98,119 @@ void whenAddedCharacterCard_thenReturnCharacterCard() throws Exception {
...
+        String existingCharacterCardWithoutId = mockMvc.perform(
+                        MockMvcRequestBuilders.post("/api/character_cards")
+                                .contentType(MediaType.APPLICATION_JSON)
+                                .content("""
+                                        {"character": "test"}
+                                        """)
+                )
+                .andExpect(status().isCreated())
+                .andExpect(content().json("""
+                        {"character": "test"}
```
Comment (WE-Kaito) [inline backend/src/test/java/de/neuefische/koheis/backend/charactercards/CharacterCardIntegrationTest.java:121]: Although it's not long, you could define this testCharacter before the tests.

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -65,6 +65,13 @@ export default function App() {
             .then(closeModalUpdate)
     }
 
+    function handleDelete(event: React.MouseEvent<HTMLButtonElement>) {
+        event.preventDefault()
+        void axios.delete("/api/character_cards/" + id)
```
Comment (WE-Kaito) [inline frontend/src/App.tsx:70]: if this void doesn't have a specific reason to be, remove it for sonar

**Review by WE-Kaito — APPROVED** [review]
😊👍 ✔

**Review by Satttoshi — APPROVED** [review]
lgtm :rocket:

File: frontend/src/App.tsx (hunk 2 above)
Comment (kohei-s) [reply frontend/src/App.tsx:70]: Thank you for your suggestion, Enrico. To be honest, I don't really understand [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) yet. So, I'm learning now how to use that 🤓 I will make improvements to the cods on my next PR 🚀

**Pull request merged**